openlane_pr_tag/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Create a new pull request to update OpenLane tag.
+
+inputs:
+  ol_tag:
+    description: OpenLane tag to use in PR
+    required: true
+  path:
+    description: Repo path
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+
+    - name: Update stable tag
+      shell: bash
+      run: |
+        cd ${{ inputs.path }}
+        echo "${{ inputs.ol_tag }}" > .github/openlane-stable-tag
+
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v4
+      with:
+        path: ${{ inputs.path }}
+        draft: true
+        title: "[BOT] Update OpenLane stable tag."
+        body: |
+          This is an automated PR.
+          See the individual commits for details.
+        commit-message: |
+          [BOT] Update OpenLane stable tag -> ${{ inputs.ol_tag }}
+        branch: openlane-tag-update-branch
```

Comment:
shouldn't we include the tag inside the branch name?

Reply:
I think this is a decision we need to make. If we use the tag on the branch name, then each new tag (i.e., each time the action is run) would be a new PR. Keeping the tag name out of the branch name the action would update an open PR.

Reply:
Make senses, I didn't realize that the actions runs would need to stack up.